src/api/AuthServer/AuthServer/Exceptions/AlreadyExistingUserException.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace AuthServer.Exceptions
+{
+    public class AlreadyExistingUserException : Exception
+    {
+        public AlreadyExistingUserException(string message) : base(message)
```

Comment:
```suggestion
        public AlreadyExistingUserException(string email) 
        : base($"User with email {email} already exists")
```

It would be cleaner in calling code, wouldn't it?

Reply:
Or `ApplicationUser user` argument and then use {user.Email}

Reply:
fixed